src/Pages/Modules/ModuleFooter.js

```diff
@@ -1,28 +1,28 @@
-import React, { Component } from 'react';
-import style from  '../../assets/styles/modules.css';
-import ModuleButton from './ModuleButton';
-import ModuleObservable from './ModuleObservable';
-import ModuleServiceBack from './ModuleServiceBack';
-import ModuleForm from './ModuleForm';
-import Notifications, {notify} from 'react-notify-toast';
-//import { triggerAsyncId } from 'async_hooks';
+import React, { Component } from 'react'
+import style from  '../../assets/styles/modules.css'
+import ModuleButton from './ModuleButton'
+import ModuleObservable from './ModuleObservable'
+import ModuleServiceBack from './ModuleServiceBack'
+import ModuleForm from './ModuleForm'
+import Notifications, {notify} from 'react-notify-toast'
+//import { triggerAsyncId } from 'async_hooks'
 
 export default class ModuleFooter extends Component {
 
   constructor(props) {
-    super(props);
-    this.state = {isChanged:false, isadding: false};
+    super(props)
```

Comment:
This was solve better in another PR. please update your code with the latest master!